### DISCUSSION TITLE
I have not yet received the benefits of the  GitHub student Developer Pack.

### DISCUSSION THREAD

**Discussion by MishuZero** [community/community#131665] [GitHub Education] [answered]
### Hi everyone,

First time due to some settings I got rejected.
Then after fixing those issues I got approval but still didn't received the pack. Is there anything else I need to do?

![image](https://github.com/community/community/assets/156060788/da586a4c-3515-4ce6-9a88-4a9160e7603a)

**Comment by davevad93** [comment, marked as answer]
Hi @MishuZero ,

The message says that your benefits will be available soon, when available the status label will turn purple and change to:

`Congrats! Your request for @MishuZero was approved and your benefits are now available`

That could take a while, usually it's a matter of a few days but sometimes it can take even a couple of weeks, the waiting time may vary depending on different factors e.g. the volume of applications.

**Reply by DuckGOGO2** [reply]
Thats great

**Reply by MishuZero** [reply]
Thank you @davevad93
